tests/integration/helpers.bash

```diff
@@ -616,3 +617,42 @@ function requires_kernel() {
 		skip "requires kernel $1"
 	fi
 }
+
+function requires_idmap_fs() {
+	local fs
+	fs=$1
+
+	# We need to "|| true" it to avoid CI failure as this binary may return with
+	# something different than 0.
+	stderr=$($FS_IDMAP "$fs" 2>&1 >/dev/null || true)
```

Comment:
In shell, `2>&1 >/dev/null` is not working as expected. You have to use `>/dev/null 2>&1`.